tests/unit/connect/render.cpp

```diff
@@ -16,8 +16,9 @@ using namespace json;
 namespace {
 
 constexpr const char *const print_file = "/usb/box.gco";
-constexpr const char *const rejected_event_printing = "{\"state\":\"PRINTING\",\"command_id\":11,\"event\":\"REJECTED\"}";
+constexpr const char *const rejected_event_printing = "{\"reason\":\"Job ID doesn't match\",\"state\":\"PRINTING\",\"command_id\":11,\"event\":\"REJECTED\"}";
```

Comment:
There are `raw string literals` in c++ which allow cleaner writing of const JSON strings, so that you don't have to escape every `"` in the string
`R"(string that doesn't need to  be escaped and i can put as many " as i want in it)"`